README.md

```diff
@@ -67,6 +67,48 @@ provided packages and install only the desired slices into the *myrootfs*
 folder, according to the slice definitions available in the
 ["ubuntu-22.04" chisel-releases branch](<https://github.com/canonical/chisel-releases/tree/ubuntu-22.04>).
 
+## Chisel support for Pro archives
```

Comment:
The title and the two paragraphs below are starting with "Chisel".  Much like in our conventions for code, this is all about Chisel, so the self-reference is typically unnecessary. For example, `## Support for Pro archives` works fine. Below, the sentence can be simplified with `"To fetch and install packages from Ubuntu Pro archives, the archive has to be ..."`, and so forth.